src/components/Editor/Editor.module.css

```diff
@@ -0,0 +1,89 @@
+:global(.cm-tooltip.playground-autocompletion-dialog),
+:global(
+    .cm-tooltip.playground-autocompletion-dialog .cm-tooltip.cm-completionInfo
```

Comment:
Seem a bit fragile, any way to provide a className?

Reply:
Same for all the selector in this file. It's a module file but actually all selectors here are global

Reply:
I thought the same thing as both of your comments
Have any suggestion how to make it better?

- about the fragile - I did provided a className but CodeMirror specificity enforced me to increase my specificity
- about the module - couldn't come up with a better idea. What do you think can be done? remember that the "root" renderer of the addon's Editor is the `Panel.tsx`, so it should load either there or on the `Editor.tsx`